frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -315,18 +314,30 @@ class MidCircuitMeasure(HybridOp):
 
     binder = qmeasure_p.bind
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.postselect = self.in_classical_tracers[-2]
-        self.reset = self.in_classical_tracers[-1]
+    # pylint: disable=too-many-arguments
+    def __init__(
+        self,
+        in_classical_tracers,
+        out_classical_tracers,
+        regions: List[HybridOpRegion],
+        reset: bool = None,
+        postselect: int = None,
+    ):
+        HybridOp.__init__(self, in_classical_tracers, out_classical_tracers, regions)
+        self.reset = reset
+        self.postselect = postselect
+        # self.mv = self.out_classical_tracers[0]
```

Comment:
What is the TODO going with that again?

Reply:
Sorry I'm not sure I understand the question.

Reply:
Do we want to simply delete that line, or use it in the future? If so, what's the intent behind this line?

Reply:
Oh I see. I'm going to remove that line as it's not related to the changes in this PR. I was prototyping the `MidMeasureMP` inheritance and forgot to remove it.